Set/Models/GameModel.swift

```diff
@@ -18,6 +18,10 @@ struct GameModel {
     var startGameDate: Date
     var previousSuccesfullMatchDate: Date?
     var previousTurnDuration: TimeInterval?
+    var availableSetsOnScreen: [[CardModel]] = []
+    var isNoMoreSetAvailable: Bool {
```

Comment:
Nice computed property, but we usually try to avoid `nagative` meaning in naming, e.g. not `isNotEnabled` but `isDisabled`, not `isNotFinished` but `isFinished`, to simplify reading somewhere else like `if !isNotFinished` (what reads as if not is not finished).
So, I would recommend name it `isMoreSetsAvailable` and where you need check `if !isMoreSetsAvailable`

Reply:
Another point, is that for me as reviewer is not clear what kinds of sets, I would name it `availableMatchesOnScreen` and `isMoreMatchesOnScreenAvailable` something like

Reply:
and from the point of code disposition I would place `availableMatchesOnScreen` next to  `cardsOnTheScreen` on line 15

Reply:
fixed